joomla/joomla-cms
# [5.0] Remove special handling of pkg_search for updates from 3.10 to 4.x

Pull Request for Issue # .

### Summary of Changes

This pull request (PR) removes the special handling for the old search component's package extension which was once added with PR #35164 for updates from 3.10 to 4.x.

The update SQL scripts have already been removed with PR #40083 . This PR here removes the rest.

The PR is one in a sequence of PR's for removing obsolete actions from script.php (and other places like here in build.php) which are not relevant anymore because updates to 5.x will have to be done from a 4.4 version, like it is now with 4.x from 3.10.

### Testing Instructions

Code review: Make sure that there is not anything related to "pkg_search" left in PHP code or SQL files or anywhere else, and that the manifest file "pkg_search.xml" is deleted on GitHub.

Optional real test: Make sure that updating the latest 4.4-dev nightly build to the update package created by drone works when using the upload & update method. Make also sure that the build.php script still works, i.e. run `php ./build/build.php --remote=HEAD --exclude-gzip --exclude-bzip2` on the branch of this PR.

### Actual result BEFORE applying this Pull Request

Obsolete manifest file "pkg_search.xml" is present in update packages, and obsolete code in script.php and build.php related to that file is present.

### Expected result AFTER applying this Pull Request

No manifest file "pkg_search.xml" is present in update packages, and no obsolete code in script.php and build.php related to that file is present.

Updating from a 4.4-dev nightly build works.

`php ./build/build.php --remote=HEAD --exclude-gzip --exclude-bzip2` still works. The update package doesn't contain the "pkg_search.xml" manifest file.

### Link to documentations
Please select:
